mne_connectivity/spectral/epochs_multivariate.py

```diff
@@ -99,35 +100,51 @@ class _EpochMeanMultivariateConEstBase(_AbstractConEstBase):
 
     name = None
     n_steps = None
+    con_scores = None
     patterns = None
     filters = None
     con_scores_dtype = np.float64
 
     def __init__(
-        self, n_signals, n_cons, n_freqs, n_times, *, store_filters=False, n_jobs=1
+        self,
+        n_signals,
+        n_cons,
+        n_freqs,
+        n_times,
+        *,
+        n_components=0,
+        store_con=True,
+        store_filters=False,
+        n_jobs=1,
     ):
```

Comment:
Added a little optimisation with a `store_con` flag, which determines whether connectivity results are actually stored.
Is default `True` for use in cases like the `spec_conn` functions, but can be set to `False` for the decomposition cases where memory does not need to be allocated for the connectivity results.